__tests__/suggestedActions.js

```diff
@@ -174,4 +174,31 @@ describe('suggested-actions command', () => {
 
     expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
   });
+
+  test('suggested action bar should reset after button click', async () => {
+    const { driver, pageObjects } = await setupWebDriver({
+      cardActionMiddleware: () => next => async ({ cardAction, getSignInUrl }) => {
+        const { type, value } = cardAction;
```

Comment:
`value` seems not used here.

```suggestion
        const { type } = cardAction;
```